lib/src/metta/runner/builtin_mods/das.rs

```diff
@@ -0,0 +1,92 @@
+use std::sync::{Arc, Mutex};
+
+use hyperon_atom::{rust_type_atom, Atom, CustomExecute, ExecError, Grounded};
+use hyperon_space::DynSpace;
```

Comment:
See comment about type below.
```suggestion
use hyperon_space::{DynSpace, ATOM_TYPE_SPACE};
```